src/index.ts

```diff
@@ -281,6 +287,27 @@ async function update() {
   tryUpdate();
 }
 
+async function getLatestVersionLink() {
+  const LATEST_VERSION_URL = "https://api.github.com/repos/Rookout/explorook/releases/latest";
+  const VERSION_REQUEST_HEADER = {headers: {"Content-Type": "application/json"}};
+  const verNum = (await (await fetch(LATEST_VERSION_URL, VERSION_REQUEST_HEADER)).json()).name;
+  return await getPlatformDownloadLink(verNum);
+}
+
+async function getPlatformDownloadLink(verNum: string) {
+  const opSys = os.platform() === "darwin" ? "latest-mac.yml" : os.platform() === "linux" ? "latest-linux.yml" : "latest.yml";
+  const fileExt = os.platform() === "darwin" ? "dmg" : os.platform() === "linux" ? "AppImage" : "exe";
```

Comment:
First, `opSys` is kind of a misleading name, the string doesn't really describe the operating system anyhow, it describes the name of a `yml` file

Second, avoid nested ternaries, they're hard to follow. Maybe make a constant object that's like:

```javascript
const LINK_PROPS_BY_OS = {
    darwin: {ymlName: 'latest-mac.yml', fileExt: 'dmg'},
    linux: {ymlName: 'latest-linux' , fileExt: 'AppImage'},
    windows: {ymlName: 'latest' , fileExt: 'exe'}
}

```

And then 

```javascript
    const { ymlName, fileExt } = LINK_PROPS_BY_OS[os.platform()]
```